public/css/main.css

```diff
@@ -51,6 +51,22 @@ a:focus {
     font-weight: bold;
 }
 
+.layerNewButtonDropdown {
+    background-color: #3f4652;
+}
+
+#layerButtonGroup {
+    margin-left: 35px;
+}
```

Comment:
It looks like you tried to center the button by using a left margin. This should be avoided. And this button does not need to be centered